Allow compiling crashpad with gcc (fetch compiler from environment variables)
--- third_party/mini_chromium/mini_chromium/build/config/BUILD.gn
+++ third_party/mini_chromium/mini_chromium/build/config/BUILD.gn
@@ -393,8 +393,8 @@
     ar = rebase_path(clang_path, root_build_dir) + "/bin/llvm-ar"
     ld = cxx
   } else {
-    cc = "clang"
-    cxx = "clang++"
+    cc = getenv("CC")
+    cxx = getenv("CXX")
     asm = cxx
     ld = cxx
 
